cogs/bot.py

```diff
@@ -478,15 +478,15 @@ async def pick(self, ctx, *, name: str):
         if result is None:
             return await ctx.send("Time's up. Aborted.")
         if result is False:
-            return await ctx.send(
+            return await ctx.reply(
                 "Since you chose not to accept the new user terms, we are unable to grant you access to Pokétwo.\n"
                 "If you wish to continue, please re-run the command and agree to our Terms of Service to continue.",
             )
 
         member = await self.bot.mongo.fetch_member_info(ctx.author)
 
         if member is not None:
-            return await ctx.send(
+            return await ctx.reply(
```

Comment:
Could you do `mention_author=False` for this because we don't want users' mentions tabs to get clogged.